### DISCUSSION TITLE
How do I change default location of my view?

### DISCUSSION THREAD

**Discussion by Ryujiho** [microsoft/vscode-discussions#410] [Extension Development QnA] [answered]
Hi, I would like to change default location of my custom view as if I had just opened my extension for the first time. I want to make the location to bottom like terminal view.

I contributed a view by using [registering a provider](https://code.visualstudio.com/api/references/vscode-api#window.registerWebviewViewProvider). It seems there are no options and commands to change the default location for extension view. In default VSCode view(ex. terminal view), there is a setting to change default location. But seems like extension doesn't provide changing default location command. Could you help me if I could use this function please?

I have done a search related to 'default location view', there were only one discussion related to. (https://github.com/microsoft/vscode-discussions/discussions/142)

**Comment by gjsjohnmurray** [comment]
I think you can contribute a view container to the Panel, and contribute your view to that view container. See https://github.com/microsoft/vscode/issues/166253#issuecomment-1314955268

**Comment by ArturoDent** [comment, marked as answer]
`panel` is one of the ViewContainer options: https://code.visualstudio.com/api/references/contribution-points#contributes.viewsContainers

```jsonc
	"viewsContainers": {
			"panel": [
				{
					"id": "editorManager",
					"title": "Editor Manager",
					"icon": "$(eye)"
				}
			]
		},
		"views": {
			"editorManager": [         // same as above id
				{
					"id": "editor-groups",
					"name": "Groups",
					"size": 1,
					"visibility": "visible"
				}
			]
		},
```
